tests/test_process.py

```diff
@@ -66,7 +66,17 @@ class TestProcessDataset:
         }
     }
 
-    def setup_method(self, syn):
+    dataset_object_gx_disabled = {
+        "neuropath_corr": {
+            "files": [{"name": "test_file_1", "id": "syn1111111", "format": "csv"}],
+            "final_format": "json",
+            "provenance": ["syn1111111"],
+            "destination": "syn1111113",
+            "gx_enabled": False,
```

Comment:
Just checking that we have a test for confirming this is the behavior when `gx_enabled` is not part of the config as well.

Reply:
Yup! Those tests say "gx_not_specified". [Example](https://github.com/Sage-Bionetworks/agora-data-tools/blob/a9b1db1e5e728248a811593ad737f7e1e7f22973/tests/test_process.py#L20)